OpenXmlFormats/Drawing/BaseStylesheet.cs

```diff
@@ -981,7 +981,7 @@ internal void Write(StreamWriter sw, string nodeName)
             {
                 foreach (CT_BlipFillProperties x in this.blipFill)
                 {
-                    x.Write(sw, "a:blipFill");
+                    x.Write(sw, "blipFill");
```

Comment:
blipFill is used in a few places. If you do want to change this, you have to change all the places that reference blipFill

Reply:
I have removed `a:`, because `CT_BlipFillProperties.Write` will perform prepending `a:`.

https://github.com/HiraokaHyperTools/npoi/blob/3756b13b55ac00bbff2d705cd5eca62534b92564/OpenXmlFormats/Drawing/ShapeEffects.cs#L2793

```cs
        internal void Write(StreamWriter sw, string nodeName)
        {
            sw.Write(string.Format("<a:{0}", nodeName));
...
```